### ISSUE TITLE
react_native_image_palette-Swift.h file not found

### ISSUE THREAD

**Issue by sohail-wayland** [someSOAP/react-native-image-palette#1]
Trying to compile my app and it fails in Xcode with the following error:

`@somesoap/react-native-image-palette/ios/ImagePalette.mm:3:9: 'react_native_image_palette-Swift.h' file not found`

My environment:
RN: 0.76.1
iOS 18.2 simulator

**Comment by sohail-wayland** [comment]
Fixed this in this PR: https://github.com/someSOAP/react-native-image-palette/pull/2

**Comment by someSOAP** [comment]
@sohail-wayland hi, thanks for your PR, but it does not seems to be the universal solution. 
The issues you've faced seems to relate to 👉 [this issue](https://github.com/callstack/react-native-builder-bob/discussions/412#discussioncomment-6352402) 👈 

Check if such import of bridging header works well in your case:
```objc
#if __has_include("react_native_image_palette-Swift.h")
#import "react_native_image_palette-Swift.h"
#else
#import <react_native_image_palette/react_native_image_palette-Swift.h>
#endif
```

**Comment by someSOAP** [comment]
Fixed in v1.1.0
